lib/blocs/liked_photos_cubit.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:reaxit/api/api_repository.dart';
+import 'package:reaxit/api/exceptions.dart';
+import 'package:reaxit/blocs.dart';
+import 'package:reaxit/models.dart';
+
+typedef LikedPhotosState = DetailState<List<AlbumPhoto>>;
```

Comment:
I think this needs pagination. Someone may have >100 liked photos, and in that case:
- the API will paginate by default, so as it is now they won't all be shown
- if you pass a high limit to override the default pagination, we create excessive load on the server

Reply:
Update: okay actually paginating it (while it would be great) also complicates the logic of un-liking a bit (you'd need to change the offsets to make sure you fetch the right page next, or you'd have to refresh the entire screen to remove a single item). So just setting a high limit is _acceptable_. But would be great to paginate if you're up for a little bit of a challenge.

Reply:
> Now I just have it so that it does not remove the photo if it is unliked, but the heart icon does go away.

That sounds nice but will actually lead to a small bug:
If I have 100 liked photos, and unlike one (e.g. number 5) on the first page, then on the backend, I'll only have 99 left. If after unliking, I try to fetch the second page (e.g. photos 30-60), that page will be shifted right by one, I would not get photo 30, because the server thinks it would have been number 29 (because 5 is no longer liked)

Reply:
A solution could be to decrease/increase the offset whenever you unlike/re-like a photo. But of course that does need proper error handling (undo the change if un/re-liking failed)

Reply:
I think it should work now.